src/vs/workbench/services/configurationResolver/common/configurationResolver.ts

```diff
@@ -22,6 +22,11 @@ export interface IConfigurationResolverService {
 	 */
 	resolveAny(folder: IWorkspaceFolder | undefined, config: any, commandValueMapping?: IStringDictionary<string>): any;
 
+	/**
+	 * Like resolveAny, but also includes variable contributed by other parts of the system.
+	 */
+	resolveAnyIncludeContributed(folder: IWorkspaceFolder | undefined, config: any, commandValueMapping?: IStringDictionary<string>): Promise<any>;
```

Comment:
Why do we have a special `resolveAnyIncludeContributed`?
I think `resolveAny` shuold always take care of contributed variables